matching-service/model/match-info-model.js

```diff
@@ -0,0 +1,25 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "./repository.js";
+
+/**
+ * Stores the information about matches made.
+ * Property: usernameOne < usernameTwo
+ */
+export const MatchInfo = sequelize.define("MatchInfo", {
+    usernameOne: {
+        type: DataTypes.STRING,
+        allowNull: false,
+    },
+    usernameTwo: {
+        type: DataTypes.STRING,
+        allowNull: false,
+    },
+    difficultyLevel: {
+        type: DataTypes.ENUM,
+        values: ["easy", "hard", "difficult"],
```

Comment:
Is this supposed to be easy, medium and hard?

Reply:
Good catch!